db/seeds.rb

```diff
@@ -1,109 +1,144 @@
+              website: Faker::Internet.url,
+              admin: true)
+end
 
-User.create company:					"Dev Man",
-						email:						"dev@dev.com",
-						password:	        "123456",
-						employees:				rand(8),
-						website:					Faker::Internet.url,
-						admin:						false
+CREATE_ONE_DEV = 1
+CREATE_ONE_DEV.times do
+  User.create(company: 'Dev Man',
+              email: 'dev@dev.com',
+              password: '123456',
+              employees: rand(8),
+              website: Faker::Internet.url,
+              admin: false)
+end
 
-business = ["Running", "Closed", "Updating"]
+BUSINESS_STATUSES = %w(Running Closed Updating).freeze
 5.times do
-    Location.create name:           Faker::Company.name,
-                  	address:        Faker::Address.street_address,
-                    city:           Faker::Address.city,
-                    postal_code:    Faker::Address.zip_code,
-                    run_status:     business.sample
+  NUMBER += 1
+  Location.create(name:           Faker::Company.name + NUMBER.to_s,
+                  address:        Faker::Address.street_address + NUMBER.to_s,
+                  city:           Faker::Address.city + NUMBER.to_s,
+                  postal_code:    Faker::Address.zip_code + NUMBER.to_s,
+                  run_status:     BUSINESS_STATUSES.sample)
 end
 
 20.times do
-    Machine.create location_id:     1+rand(5)
+  Machine.create location_id:     1 + rand(5)
+end
+
+LIST_OF_STATUSES = %w(under_review
+                      rejected
+                      unreleased
+                      released
+                      incompatible).freeze
+LIST_OF_STATUSES.each do |s|
+  Status.create name: s
 end
 
-["Adventure", "Action", "Strategy", "Multi-player", "Co-op", "Single-player", "Sport", "Racing"].each do |t|
-    Tag.create name: t
+LIST_OF_TAGS = %w(Adventure
+                  Action
+                  Strategy
+                  Multi-player
+                  Co-op
+                  Single-player
+                  Sport
+                  Racing).freeze
+LIST_OF_TAGS.each do |t|
+  Tag.create name: t
 end
 
 100.times do
-    Game.create({title: Faker::Hipster.word,
-                  description: Faker::Hipster.paragraph,
-                        user_id: 1+rand(22),
-                        cpu: rand(100)+1,
-                        gpu: rand(50)+1,
-                        ram: 4+rand(17),
-                        size: 1+ rand(4),
-                        aasm_state: ["Released to arcade", "Not released", "Game under review", "Game not uploaded, it is not compatible with the system", "Rejected"].sample })
+  NUMBER += 1
+  Game.create(title: Faker::Hipster.word + NUMBER.to_s,
+              description: Faker::Hipster.paragraph + NUMBER.to_s,
+              user_id: 1 + rand(22),
+              cpu: rand(100) + 1,
+              gpu: rand(50) + 1,
+              ram: 4 + rand(17),
+              size: 1 + rand(4),
+              status_id: 1 + rand(5))
```

Comment:
This is unnecessary (See above). Replace with AASM_state. The five possible aasm_status are:
["under_review", "rejected", "unreleased", "released", "incompatible"]